colabs/wandb-artifacts/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,433 @@
+      "metadata": {
+        "id": "dX5cFz_XCd5h"
+      },
+      "source": [
+        "# Set Up"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "q2EeMdcpC7Dl"
+      },
+      "source": [
+        "In order to use Weights & Biases, you will need the `wandb` package installed. You can install it as follows within Colab."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "nwEF5d7NChdB"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install wandb"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "X7V-9P5-mAH6"
+      },
+      "source": [
+        "Once it is installed, the next step is to import it into your script or notebook with `import wandb`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "E7-3aU-zmAH6"
+      },
+      "outputs": [],
+      "source": [
+        "import wandb"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "arOKvIXFmAH7"
+      },
+      "source": [
+        "We also need to authenticate to the Weights & Biases server. There are various ways of doing this, including for [remote or non-interactice workflows](https://docs.wandb.ai/guides/track/environment-variables), but given this is running interactively, we can use `wandb.login()`.\n",
+        "\n",
+        "If we are not already authenticated, a link will appear which you can use to do so."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Y6lztcX5mAH7"
+      },
+      "outputs": [],
+      "source": [
+        "wandb.login()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vp9Zhzp5Hvt1"
+      },
+      "source": [
+        "# Create An Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3xEsZzOEnPFY"
+      },
+      "source": [
+        "The general workflow for creating an Artifact is:\n",
+        "\n",
+        "\n",
+        "1.   Intialize a run.\n",
+        "2.   Create an Artifact.\n",
+        "3.   Add a any files or directories to the new Artifact that you want to track and version.\n",
+        "4.   Log the artifact in the W&B platform.\n",
+        "\n",
+        "The most straightforward way of accomplishing this is the second line of code in the example below, which will log, track and version a new dataset."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Vndk289hgVVK"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "run.log_artifact(artifact_or_path=\"/content/sample_data/mnist_test.csv\", name=\"my_first_artifact\", type=\"dataset\")\n",
```

Comment:
Notebooks need to be executable...we'll want to use a real dataset before merging this in.